.github/CODEOWNERS

```diff
@@ -54,6 +54,7 @@
 /storagetransfer/**/*                  @GoogleCloudPlatform/cloud-storage-dpes @GoogleCloudPlatform/python-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers
 # ---* Infra DB
 /alloydb/**/*                          @GoogleCloudPlatform/alloydb-connectors-code-owners @GoogleCloudPlatform/python-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers
+/alloydb/notebooks/*                   @GoogleCloudPlatform/alloydb-connectors-code-owners @GoogleCloudPlatform/python-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers tabathal@google.com kpushkar@google.com
```

Comment:
Do you have a group that we can use instead of individuals?

As well, these should be GitHub usernames instead

Reply:
+1 to above comment. Preferable a TeamSync managed team like https://github.com/orgs/GoogleCloudPlatform/teams/cloud-samples-reviewers

Reply:
@tabbyl21 I've emailed you the instructions to set this up